packages/web3-eth-accounts/src/common/utils.ts

```diff
@@ -0,0 +1,619 @@
+		// - Merge hardfork has to be placed just after genesis if any of the genesis hardforks make CL
+		//   necessary for e.g. withdrawals
+		const mergeConfig = {
+			name: Hardfork.Merge,
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment
+			ttd: config.terminalTotalDifficulty,
+			// eslint-disable-next-line no-null/no-null
+			block: null,
+		};
+
+		// Merge hardfork has to be placed before first hardfork that is dependent on merge
+		const postMergeIndex = params.hardforks.findIndex(
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+			(hf: any) => forkMap[hf.name]?.postMerge === true,
+		);
+		if (postMergeIndex !== -1) {
+			params.hardforks.splice(postMergeIndex, 0, mergeConfig as unknown as ConfigHardfork);
+		} else {
+			params.hardforks.push(mergeConfig as unknown as ConfigHardfork);
+		}
+	}
+
+	const latestHardfork = params.hardforks.length > 0 ? params.hardforks.slice(-1)[0] : undefined;
+	params.hardfork = latestHardfork?.name;
+	params.hardforks.unshift({ name: Hardfork.Chainstart, block: 0 });
+
+	return params;
+}
+
+/**
+ * Parses a genesis.json exported from Geth into parameters for Common instance
+ * @param json representing the Geth genesis file
+ * @param name optional chain name
+ * @returns parsed params
+ */
+export function parseGethGenesis(json: any, name?: string, mergeForkIdPostMerge?: boolean) {
+	try {
+		if (['config', 'difficulty', 'gasLimit', 'alloc'].some(field => !(field in json))) {
+			throw new Error('Invalid format, expected geth genesis fields missing');
+		}
+		if (name !== undefined) {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, no-param-reassign
+			json.name = name;
+		}
+		return parseGethParams(json, mergeForkIdPostMerge);
+	} catch (e: any) {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/restrict-template-expressions
+		throw new Error(`Error parsing parameters file: ${e.message}`);
+	}
+}
+
+/**
+ * Pads a `String` to have an even length
+ * @param value
+ * @return output
+ */
+export function padToEven(value: string): string {
+	let a = value;
+
+	if (typeof a !== 'string') {
+		throw new Error(`[padToEven] value must be type 'string', received ${typeof a}`);
+	}
+
+	if (a.length % 2) a = `0${a}`;
+
+	return a;
+}
+
+/**
+ * Converts an `Number` to a `Buffer`
+ * @param {Number} i
+ * @return {Buffer}
+ */
+export const intToBuffer = function (i: number) {
+	const hex = intToHex(i);
+	return Buffer.from(padToEven(hex.slice(2)), 'hex');
+};
+
+/**
+ * Attempts to turn a value into a `Buffer`.
+ * Inputs supported: `Buffer`, `String` (hex-prefixed), `Number`, null/undefined, `BigInt` and other objects
+ * with a `toArray()` or `toBuffer()` method.
+ * @param v the value
+ */
+export const toBuffer = function (v: ToBufferInputTypes): Buffer {
```

Comment:
Could this replace our [bytesToBuffer](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L84)?

Reply:
toBuffer supports `Buffer`, `String` (hex-prefixed), `Number`, null/undefined, `BigInt` and other objects as an input.
our `bytesToBuffer` supports only `Buffer`